docker/Dockerfile

```diff
@@ -5,3 +5,17 @@ WORKDIR /requirements
 RUN pip install -r requirements.txt -r requirements_dev.txt
 RUN mkdir /code
 WORKDIR /code
+
+
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
```

Comment:
myślałem, że zrobimy to w envie ale jeżeli instalujemy to bezpośrednio w obrazie dockera to lepiej by było przerzucić całość na górę pliku (zaraz po FROM python....) docker działa na zasadzie warstw, i dopiero jak coś się zmieni podczas budowania to reszte przebudowuje (chyba że wymusisz to na nim)

Przykładowo, dockerfile wykonuje operacje A, B, C, D i E - jeżeli zmienisz coś w C to docker przebuduje od nowa C, D i E natomiast A i B wezmie stare - czyli jeżeli kolejność ma małe znaczenie a C jest często modyfikowane to lepiej C dać na koniec instrukcji: A, B, D, E, C

U nas mogą być często przebudowywane zależności (requirements.txt), natomiast webdriver'a nie będzie chyba potrzeby ściągać ponownie - chyba, że się mylę i trzeba go aktualizować często też lub nawet częściej niż plik requirements - wtedy okej, wszystko jest w porządku :P